__tests__/app.test.js

```diff
@@ -166,19 +166,68 @@ describe("test4.get /api/articles/:article_id", () => {
   });
 });
 
+// KIRIL
+// describe("test5 get /api/users", () => {
+//   test("status:200, responds with an array of test users objects", () => {
+//     return request(app)
+//       .get("/api/users")
+//       .expect(200)
+//       .then((response) => {
+//         const {
+//           body: { users },
+//         } = response;
+//         expect(users).toBeInstanceOf(Array);
+//         expect(users).toHaveLength(4);
+//         users.forEach((user) => {
+//           expect(user).toEqual(
+//             expect.objectContaining({
+//               username: expect.any(String),
+//               name: expect.any(String),
+//               avatar_url: expect.any(String),
+//             })
+//           );
+//         });
+//         expect(users).toEqual([
+//           {
+//             avatar_url:
+//               "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg",
+//             name: "jonny",
+//             username: "butter_bridge",
+//           },
+//           {
+//             avatar_url:
+//               "https://avatars2.githubusercontent.com/u/24604688?s=460&v=4",
+//             name: "sam",
+//             username: "icellusedkars",
+//           },
+//           {
+//             avatar_url:
+//               "https://avatars2.githubusercontent.com/u/24394918?s=400&v=4",
+//             name: "paul",
+//             username: "rogersop",
+//           },
+//           {
+//             avatar_url:
+//               "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+//             name: "do_nothing",
+//             username: "lurker",
+//           },
+//         ]);
+//       });
+//   });
+// });
+

```

Comment:
Move any stuff you want to keep into a gitignored file before push so you don't have any ugly stuff here. Remember, you want your Main branch to be working code which you're happy would survive the enduser, and/or could be scrutinised by your whole organisation/potential employers.
In continuous-deployment your Main would literally be what get updated to the client.